helm/cluster-api-core/templates/webhook-deployment.yaml

```diff
@@ -1,65 +1,63 @@
 apiVersion: apps/v1
```

Comment:
Moved this from the child `webhook` directory so it would be installed by helm. Alternatively this could be a subchart, but that felt like too much complexity.